### PR TITLE
logging changes and renamings

### PR DESCRIPTION
**Pull request by QuiteStochastic**
Thank you for contributing to Coda! Please see `CONTRIBUTING.md` if you haven't
yet. In that doc, there are more details around how to start our CI.

Explain your changes here.

Explain how you tested your changes here.

Checklist:

- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them:

Closes #0000
Closes #0000

### CODE REVIEW

File: src/lib/integration_test_cloud_engine/stack_driver_log_engine.ml [hunk 1]
```diff
@@ -693,7 +694,8 @@ let create ~logger ~(network : Kubernetes_network.t) ~on_fatal_error =
       ~handle_result:(fun result ->
         let open Initialization_query.Result in
         let open Malleable_error.Let_syntax in
-        [%log info] "Handling initialization log for node \"%s\"" result.pod_id ;
+        [%log debug] "Handling initialization log for node \"%s\""
```
Comment (mrmr1993) [inline src/lib/integration_test_cloud_engine/stack_driver_log_engine.ml:697]: Nit: use `~metadata`, e.g.

```ocaml
[%log debug] "Handling initialization log for node $node_id"
  ~metadata: [("node_id", `String result.pod_id)]
```

File: src/lib/integration_test_cloud_engine/stack_driver_log_engine.ml [hunk 2]
```diff
@@ -598,7 +598,7 @@ let rec pull_subscription_in_background ~logger ~subscription_name
       let%bind logs = Subscription.pull subscription in
       Malleable_error.List.map logs ~f:parse_subscription)
   in
-  [%log info] "Pulling %s subscription" subscription_name ;
+  [%log debug] "Pulling %s subscription" subscription_name ;
```
Comment (mrmr1993) [inline src/lib/integration_test_cloud_engine/stack_driver_log_engine.ml:601]: Nit: use `~metadata`, e.g.

```ocaml
[%log debug] "Pulling subscription $subscription_name"
  ~metadata:[("subscription_name", `String subscription_name)]
```

File: src/app/test_executive/block_production_test.ml [hunk 3]
```diff
@@ -21,10 +21,14 @@ module Make (Engine : Engine_intf) = struct
...
     let open Network in
     let open Malleable_error.Let_syntax in
+    let logger = Logger.create () in
     let block_producer = List.nth_exn network.block_producers 0 in
     let%bind () = Log_engine.wait_for_init block_producer log_engine in
     let%map _ =
       Log_engine.wait_for ~blocks:1 ~timeout:(`Slots 30) log_engine
     in
+    [%log info]
+      "block_production_term: cannot remove '.coda-config': Device or \
```
Comment (mrmr1993) [inline src/app/test_executive/block_production_test.ml:31]: This log message is unusual, did it get garbled somehow?

File: src/lib/integration_test_cloud_engine/stack_driver_log_engine.ml [hunk 4]
```diff
@@ -743,12 +747,16 @@ let create ~logger ~(network : Kubernetes_network.t) ~on_fatal_error =
...
   in
   let cancel_background_tasks () =
+    let open Deferred.Let_syntax in
     if not (Ivar.is_full cancel_background_tasks_ivar) then
       Ivar.fill cancel_background_tasks_ivar () ;
-    Deferred.all_unit
-      [ errors_task_finished
-      ; initialization_task_finished
-      ; transition_frontier_diff_application_finished ]
+    let%bind () =
```
Comment (mrmr1993) [inline src/lib/integration_test_cloud_engine/stack_driver_log_engine.ml:753]: Nit: `let%map` and remove the `Deferred.return` below

File: src/app/test_executive/bootstrap_test.ml [hunk 5]
```diff
@@ -36,7 +37,11 @@ module Make (Engine : Engine_intf) = struct
     in
     let%bind () = Node.start ~fresh_state:true node_b in
     let%bind () = Log_engine.wait_for_init node_b log_engine in
-    Log_engine.wait_for_sync [node_a; node_b]
-      ~timeout:(Time.Span.of_ms (15. *. 60. *. 1000.))
-      log_engine
+    let%bind () =
```
Comment (mrmr1993) [inline src/app/test_executive/bootstrap_test.ml:40]: Nit: `let%map` and remove the `return` below

**Review by mrmr1993 — COMMENTED** [review]
Looks good apart from a few nits. However we'll probably want to change the weird log message (or at least add a comment explaining it) before merging.

File: src/app/test_executive/block_production_test.ml (hunk 3 above)
Comment (QuiteStochastic) [reply src/app/test_executive/block_production_test.ml]: yeah i mis-copy pasted 😅

**Review by mrmr1993 — APPROVED** [review]

**Pull request merged**